src/pages/Auth/Login.jsx

```diff
@@ -0,0 +1,62 @@
+import { NavBar } from "../../components";
+import { useAuth } from "../../contexts/auth-context"
+import {useNavigate} from "react-router-dom"
+import { useState } from "react";
+import "./Auth.css";
+
+export const Login=()=>{
+    const [loginForm,setLoginForm]= useState({email:"",password:""});
+   
+    const {loginHandler} = useAuth();
+    const navigate= useNavigate();
+
+    return (
+      <>
+       <NavBar/>
+        
+        <div className="auth-container flex-row">
+          <div className="auth-card flex-col">
+            <h1 className="auth-heading">Login</h1>
+            <p>Please enter your valid email and password.</p>
+            <div className="auth-input flex-row">
+              <label for="" className="input-label"><strong>Email</strong></label>
+              <input
+                type="text"
```

Comment:
For email input, you can use the type="email" it gives you email validation also